src/types.ts

```diff
@@ -199,33 +199,104 @@ export enum PgType {
 export type MigrationDirection = 'up' | 'down'
 
 export interface RunnerOptionConfig {
+  /**
+   * The table storing which migrations have been run.
+   */
   migrationsTable: string
+  /**
+   * The schema storing table which migrations have been run.
+   *
+   * (defaults to same value as `schema`)
+   */
   migrationsSchema?: string
+  /**
+   * The schema on which migration will be run.
+   *
+   * @default 'public'
+   */
   schema?: string | string[]
+  /**
+   * The directory containing your migration files.
+   */
   dir: string
+  /**
+   * Check order of migrations before running them.
+   */
   checkOrder?: boolean
+  /**
+   * Direction of migration-run.
+   */
   direction: MigrationDirection
+  /**
+   * Number of migration to run.
+   */
   count: number
+  /**
+   * Treats `count` as timestamp.
+   */
   timestamp?: boolean
+  /**
+   * Regex pattern for file names to ignore (ignores files starting with `.` by default).
+   */
   ignorePattern?: string
+  /**
+   * Run only migration with this name.
+   */
   file?: string
   dryRun?: boolean
```

Comment:
skipped this one? :thinking:

Reply:
There is no description text to copy


https://github.com/salsita/node-pg-migrate/blob/master/docs/api.md
![Screenshot_20210913_140335](https://user-images.githubusercontent.com/7195563/133080238-fc2ce467-f6cf-471a-9469-9a9bce470612.jpg)